internal/engine/experiment/tor/tor_test.go

```diff
@@ -378,18 +378,18 @@ func TestDefautFlexibleConnectOBFS4(t *testing.T) {
 	)
 	ctx, cancel := context.WithCancel(context.Background())
 	cancel()
-	tk, err := rc.defaultFlexibleConnect(ctx, wrapTestingTarget(staticTestingTargets[0]))
-	if err == nil {
-		t.Fatal("expected an error here")
+	tk, failure := rc.defaultFlexibleConnect(ctx, wrapTestingTarget(staticTestingTargets[0]))
+	if failure == nil {
+		t.Fatal("expected a failure here")
 	}
-	if err.Error() != "interrupted" {
+	if *failure != "interrupted" {
 		t.Fatal("not the error we expected")
 	}
-	if tk.Connects == nil {
+	if tk.TCPConnect == nil {
 		t.Fatal("expected connects data here")
 	}
-	if tk.NetworkEvents == nil {
-		t.Fatal("expected network events data here")
+	if tk.NetworkEvents != nil {
+		t.Fatal("expected no network events data here")
```

Comment:
Here's the main change since before: `measurex` only includes the TCPConnect event into the list of TCP connect attempts and _does not_ also include it inside the network events. (I think this change will not have any significant impact on the data processing pipeline, since it ignores network events 🤷.)